packet.go

```diff
@@ -207,7 +207,7 @@ func (p *sshFxInitPacket) UnmarshalBinary(b []byte) (err error) {
 		return err
```

Comment:
I personally don't like named returns or naked returns at all, outside of for documentation purposes in `interface` declarations.

I'd be happy to remove them throughout in a different PR, if you agree.